bin/synapse_index.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+import hashlib
+import os
+import re
+import sys
+
+import synapseclient
+from synapseclient.models import File
+
+
+def compute_md5_checksum(file: str) -> str:
```

Comment:
Looking at the code while I can. 

You could directly use https://github.com/Sage-Bionetworks/synapsePythonClient/blob/83a2a5aadb7f7aa3be689c6a3dd49042f9208d7b/synapseclient/core/utils.py#L88 without the need to write your own.

Reply:
The function that you linked does not exist in the latest version of the client. None of the other functions in the client work to replace exactly what this `compute_md5_checksum` si doing.

Reply:
Ah. It might only be available when 4.2.0 is released then

Reply:
I see. I can leave a note to bump the version and implement this later.

Reply:
Version 4.2.0 was released today so I was actually able to do this in this PR. [Link](https://tower-dev.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/example-dev-project/watch/euJuwJVuD36MH) to new run